Gruntfile.js

```diff
@@ -76,7 +76,10 @@ module.exports = function (grunt) {
         stylus: {
             minerva: {
                 files: [{
-                    src: ['<%= plugin.minerva.external %>/stylesheets/main.styl'],
+                    src: [
+                        '<%= plugin.minerva.external %>/stylesheets/main.styl',
+                        '<%= plugin.minerva.root %>/node_modules/colorbrewer/colorbrewer.css'
```

Comment:
This would be better in some target that runs on grunt init, similar with the js below.